packages/web-twig/src/Resources/components/TextFieldBase/TextFieldBase.twig

```diff
@@ -124,6 +129,11 @@
             />
         {% endif %}
     {% endif %}
+    {% if _helperText %}
+        <span class="{{ _helperTextClassName }}">
```

Comment:
```suggestion
        <div class="{{ _helperTextClassName }}">
```
in `web` and `web-react` we use `div`. Let's unify it. Please update snapshots as well.

Reply:
fixed